### PR TITLE
Easter eggs

### PR DESCRIPTION
**Pull request by JohnBinning**
## What's this PR do?

I added some easter eggs, unique place icons to the map.

I'm a bit worried that the setUrl function is too long.

## Pre-Merge TODOs
 - [x ] Screenshots Included
 - [ x] Tests Added
  - added tests for new url functions
 - [ ] 2 PR reviews completed

## Where should the reviewer start?

At the containerHelper 

## Screenshots (if appropriate)
screen shot 2017-06-06 at 10 05 25 pm

## What gif best describes this PR or how it makes you feel?
easter eggs are

### CODE REVIEW

File: app/assets/Components/App/App.js [hunk 1]
```diff
@@ -55,6 +55,9 @@ class App extends Component {
               <div className='underline'></div>
             </div>
            </button>
+           <footer>
+              <p className='copyright'>&#169; 2017 John Binning</p>
```
Comment (letakeane) [inline app/assets/Components/App/App.js:59]: NEAT IDEA 😁

**Comment by lauraturk** [comment]
reviewing NO code: ❤️ 💖 😍 💖 ❤️  the easter eggs.

File: app/assets/Helpers/MapContainer/ContainerHelper.js [hunk 2]
```diff
@@ -21,6 +24,56 @@ export const setFilter = (app) => {
...
+      if(formattedAddress.includes('National Park')) {
+        url = MarkerUrls.notCompletedNP
+      }
+      break
+
+    default:
+      url = MarkerUrls.notCompletedDefault
+  }
+
+  if (region === "China" || region === 'Beijing' || region === 'Balboa Park') {
```
Comment (letakeane) [inline app/assets/Helpers/MapContainer/ContainerHelper.js]: Maybe this if block could be broken out into a separate function and just called here. Like 'setEasterEggPin' or something?

File: app/assets/Helpers/MapContainer/MapContainerHelper.test.js [hunk 3]
```diff
@@ -16,8 +19,20 @@ describe('MapContainer helpers', () => {
...
+  })
 
+  it('should set the url', () => {
+    expect(setUrl(MockMarkers.Chengdu)).toEqual('/assets/images/da.png')
+    expect(setUrl(MockMarkers.DenverCompletedNotSelected)).toEqual('/assets/images/completePin.png')
+    expect(setUrl(MockMarkers.DenverNotCompletedSelected)).toEqual('/assets/images/incompHover.png')
+    expect(setUrl(MockMarkers.Sherlock)).toEqual('/assets/images/Sherlock.png')
+    expect(setUrl(MockMarkers.Everest)).toEqual('/assets/images/mountain.png')
+    expect(setUrl(MockMarkers.AcadiaNotCompleted)).toEqual('/assets/images/park_blank.png')
+    expect(setUrl(MockMarkers.AcadiaCompleted)).toEqual('/assets/images/park_blank_comp.png')
```
Comment (letakeane) [inline app/assets/Helpers/MapContainer/MapContainerHelper.test.js:35]: You're so on top of testing! It's great!

**Review by letakeane — COMMENTED** [review]
Love the easter eggs! Super creative :)

File: app/assets/Helpers/MapContainer/ContainerHelper.js (hunk 2 above)
Comment (lauraturk) [reply app/assets/Helpers/MapContainer/ContainerHelper.js]: I like Leta's idea, also could you make it more dynamic using bracket notation? It looks like completed vs notCompleted are the only differences?

File: app/assets/Helpers/MapContainer/MockMarkers.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    formattedAddress: "Acadia National Park"
+  },
+  AcadiaCompleted: {
+    completed: true,
+    selected: false,
+    region: 'Maine',
+    dreamLocation: "Acadia National Park",
+    formattedAddress: "Acadia National Park"
+  }
+}
```
Comment (ghost) [inline app/assets/Helpers/MapContainer/MockMarkers.js:51]: I'm digging how all this data is broken out into objects and passed into your helper file.

**Comment by ghost** [comment]
Easter eggs for days 👍

File: app/assets/Components/App/App.js (hunk 1 above)
Comment (JohnBinning) [reply app/assets/Components/App/App.js:59]: yeah I stole it from one of Jack's  PRs 😉

File: app/assets/Helpers/MapContainer/MapContainerHelper.test.js (hunk 3 above)
Comment (anderswood) [reply app/assets/Helpers/MapContainer/MapContainerHelper.test.js:35]: nice testing!

**Comment by JohnBinning** [comment]
@joshuajhun, @Tman22, or @rrgayhart
Have had two people give reviews.

**Pull request merged**